### DISCUSSION TITLE
How to automatically load webpage content for inactive tabs created by CreateNewTab without requiring the tab to be selected, based on the description provided?

### DISCUSSION THREAD

**Discussion by Amomo-CN** [cefsharp/CefSharp#4841] [General]
How to automatically load webpage content for inactive tabs created by CreateNewTab without requiring the tab to be selected, based on the description provided?

private void MainWindowLoaded(object sender, RoutedEventArgs e)
    {
        

        CreateNewTab("http://192.168.10.173:", false); 
        CreateNewTab("http://192.168.10.209:8080", false); 
CreateNewTab("http://192.168.100.216:8081/", false);

    }

Currently, a single, unnamed empty tab control is created, which only loads webpage content when clicked. How can automatic loading of the content be achieved without requiring a click?

**Comment by amaitland** [comment]
Are you using WinForms or WPF?
